src/main/java/org/apache/cassandra/sidecar/config/yaml/ServiceConfigurationImpl.java

```diff
@@ -174,7 +174,7 @@ public int port()
      * {@inheritDoc}
      */
     @Override
-    @JsonProperty(value = REQUEST_IDLE_TIMEOUT_MILLIS_PROPERTY, defaultValue = DEFAULT_REQUEST_IDLE_TIMEOUT_MILLIS + "")
```

Comment:
we don't have builders for all classes, I wonder what's the side effect to remove defaultValue for classes that don't have builders?

Reply:
Right. For those implementations, there is a no-args constructor defined with the default values for the fields. For example, `CassandraInputValidationConfigurationImpl`. 
Jackson always create an instance with such constructor and set the fields, during deserialization. So the default value is already present when the field is absent in the json text